logstash-core/lib/logstash/pipeline.rb

```diff
@@ -176,8 +179,10 @@ def stopped?
   end
 
   def start_workers
-    @inflight_batches = {}
-
+    @filter_queue_client.set_events_metric(metric.namespace([:stats, :events]))
```

Comment:
I prefer to use the set_* idiom, it allows us to change the arguments e.g. `set_events_metric(metric, :stats)` that infers namespace([:stats, :events]).